### DISCUSSION TITLE
Help! Where are new damage types stored?

### DISCUSSION THREAD

**Discussion by darthlunga** [Larkinabout/fvtt-custom-dnd5e#36] [Q&A]
Foundry v11
DND5E 3.0.3

This may sound a little convoluted but I am quite a novice at all this. First, great mod. Enjoying it thus far.

I created a new damage type called divine. It works. Meaning in an items details, divine shows up as a type along with all the others. I use midi for workflow and it rolls it and applies the damage. All good there.

Now I am trying to create an effect so I can apply this special damage by just applying an effect to a token. For all the other types, (thunder, cold, etc) I just use system.bonuses.mwak.damage then put say +5(thunder] in the effect value. When I type in +5[divine] it doesn't seem to recognize it and it is not added to the roll. 

I am asking "where are they stored" because my guess is it's not listed under mwak.damage? Is there another repository? Idk. Help? lol

**Comment by Larkinabout** [comment]
It looks like you had this discussion on Posney's Foundry Automation Discord and there is a possible fix being looked into for MidiQoL. I'll also make the key visible on the Configure Damage Types menu, so that users can change the key to whatever they like.
